pages/docs/castor/rao-steps.md

```diff
@@ -40,11 +40,34 @@ Overloads potentially reached on this perimeter are to be solved with preventive
+That's why regardless of the impacts on CNECs, automatic remedial actions are activated if their specific activation conditions are met (eg. after a given contingency).
+In CASTOR, this simulation is carried out in two stages:
+
+Here is a detailed step-by-step of automatic remedial actions’ application in CASTOR:
+1. First, all automatic remedial actions for which there is not set-point to compute are applied on the network. These remedial actions are called [network actions](/docs/input-data/crac/json#network-actions). They include topological actions, injection set-points, PST set-points, switch pairs.
+
+2. Then, automatic range actions are applied one by one, as long as some of the perimeter's CNECs are overloaded. The speed of the remedial actions determines the order in which they are applied : the fastest range actions are simulated first. Aligned range actions are simulated simultaneously.
+
+Automatic range actions' setpoint is computed using the results of a sensitivity analysis computation, during which the sensivity $$\sigma$$ of a range action on a cnec is computed. By focusing on the worse overloaded CNEC, we can compute the automatic range action's optimal setpoint to relieve that CNEC with the following formula:
+\begin{equation}
+A_{optimal} = A_{current} + {sign}{(F(c)}) * \frac{\min(0, margin(c))}{\sigma}
+\end{equation}
+
+| Name                           | Symbol            | Details                                                                                                               | Type                | Index                                            | Unit                                                      | Lower bound           | Upper bound           |
+|--------------------------------|-------------------|-----------------------------------------------------------------------------------------------------------------------|---------------------|--------------------------------------------------|-----------------------------------------------------------|-----------------------|-----------------------|
+| Flow                           | $$F(c)$$          | flow of FlowCnec $$c$$                                                                                                | Real value          | One variable for every element of (FlowCnecs)    | MW                                                        | $$-\infty$$           | $$+\infty$$           |
+| RA setpoint                    | $$A$$             | setpoint of RangeAction                                                                          | Real value          | One variable for every element of (RangeActions) | Degrees for PST range actions; MW for other range actions | Range lower bound[^1] | Range upper bound[^1] |
+
+
+Range actions include PSTs and HVDCs. As soon as all CNECs are secure, we stop applying range actions. 
```

Comment:
Maybe add the fact that if the range action is not enough to relieve a cnec, it is pushed to it's extremum.
Also once the worse overloaded cnec is relieved, we switch to the next worse one and try relieve that one. And if we have to change direction for the setpoint of the range action we go to the next range action. (For instance if we need to increase the tap number of a PST from 0 to 5 to solve the worse cnec, and then go back down to 3 for the next PST, then we stay at tap 5 and go to the next range action) (maybe a diagram can help)